cluster.yml

```diff
@@ -0,0 +1,21 @@
+---
+- cluster: "Smoke"
+  ambari_server: "ambari.smoke.vpc.rgops.com"
+  datanode:
+    - NODEMANAGER
+    - HBASE_CLIENT
+    - HCAT_CLIENT
+    - HDFS_CLIENT
+    - TEZ_CLIENT
+    - METRICS_MONITOR
+    - MAPREDUCE2_CLIENT
+    - SLIDER
+    - SQOOP
+    - YARN_CLIENT
+
```

Comment:
I would probably rename this, just so that it doesn't get confused with the hodor "clusters.yml" file. Nothing really wrong here, except for the possible confusion factor with our internal tooling